modules/app/components/ResourceBox.tsx

```diff
@@ -100,16 +100,16 @@ const resources: Record<ResourceType, Resource> = {
         url: 'https://manual.makerdao.com/delegation/overview/what-is-delegation'
       },
       {
-        linkTitle: "What's a Constitutional Voter Committee(CVC)?",
-        url: 'https://mips.makerdao.com/mips/details/MIP101#5-constitutional-voter-committees-cvcs-'
+        linkTitle: "What's an Aligned Voter Committee(AVC)?",
+        url: 'https://mips.makerdao.com/mips/details/MIP101#2-5-aligned-voter-committees-gov5'
       },
       {
         linkTitle: 'What is the Maker Constitution?',
```

Comment:
should this have a different title? The MIP101 title is Maker Atlas Immutable Alignment Artifact

Reply:
Good catch!